tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2015",
     "moduleResolution": "node",
-    "module": "CommonJS",
+    "module": "nodenext",
```

Comment:
you sure about this one? Not sure if this breaks old note versions like 12.

Reply:
I'm not that sure, but comparing the output file it's the same.

As for node12, it's already not supported by Apollo itself, then we're 14 and up.